cluster/gce/gci/node.yaml

```diff
@@ -19,6 +19,7 @@ write_files:
       ExecStartPre=/bin/bash -c 'OPT=""; if curl --help | grep -q -- "--retry-connrefused"; then OPT="--retry-connrefused"; fi; /usr/bin/curl --fail --retry 5 --retry-delay 3 $OPT --silent --show-error -H "X-Google-Metadata-Request: True" -o /home/kubernetes/bin/configure.sh http://metadata.google.internal/computeMetadata/v1/instance/attributes/configure-sh'
       ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/configure.sh
       ExecStart=/home/kubernetes/bin/configure.sh
+      StandardOutput=journal+console
```

Comment:
just curiosity - would journal and console be ACL'd the same way? In other words, would redirecting to console expose logs to more roles than before and can it be considered an issue?

Reply:
At least in GCE (where this is applied), journal and console are viewable by anyone with at least compute viewer permission.